sys/key_rev_value.yaml

```diff
@@ -12,9 +12,11 @@ paths:
     get:
       operationId: listRevisions
 
-  /{bucket}/{key}{/revision}{/tid}:
+  /{bucket}/{key}/{revision}: &bucket_key_revision
```

Comment:
Where did the 'key' with no revision go?

Reply:
Done.